build-win64.sh

```diff
@@ -103,44 +104,46 @@ build()
 
 dist_install()
 {
-    msg  "Installing Dino in '$DIST_DIR'!"
+    _dist_arg=${1:-$DIST_DIR}
+
+    msg  "Installing Dino in '$_dist_arg'!"
     make install
```

Comment:
Here `make install` is called and in the script for Meson `ninja install`. Does this work correctly?

Reply:
Yes. This is done to call --dep-install from meson build script but providing target directory explicitly. By default (without any args) it (build-win64.sh) will use the one provided at the build-win64.sh script. so later it is called from build-meson-windows.sh as
```bash
  bash build-win64.sh $DIST_DIR
```
whereas `$DIST_DIR` as `win64-meson`.

Reply:
It  will trigger the  cmake  again.. will fix it.

Reply:
fixed. moved `make install`  to the `build` function

Reply:
This breaks the logic of the script's behavior. I propose to combine these scripts into one as part of this task.

Reply:
I'd argue.. Combining build specific command (e.g. `make install`) with generic dist-copy is incorrect. It is better to keep them separated, thus making script usable for both cmake-based  and meson-based builds.

Anyway, I'll combine scripts into one, as it seems to be the required here.